cinema_project/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for cinema_project1 project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'cinema_project.settings')
```

Comment:
Your project should be one-styled, don't use double and single quotes at the same time. Double quotes are preferred. Consider changing the single quotes to double quotes in this line.